src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2059,7 +2059,40 @@
      **/
     public static final int RPAREN = GeneratedJavaTokenTypes.RPAREN;
     /**
-     * The {@code this} keyword.
+     * The {@code this} keyword use to refer the current object.
+     * This can also be used to call the constructor.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * this.name = name;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--EXPR -&gt; EXPR
```

Comment:
remove `|` it is not required as it not lead to anywhere.

Reply:
this is connected with `SEMI`

Reply:
Please see https://github.com/checkstyle/checkstyle/pull/9899

Reply:
Resolved.